client/app/iplb/vrack/iplb-vrack.html

```diff
@@ -50,7 +50,7 @@
         <oui-column data-title="'iplb_vrack_list_col_subnet' | translate" data-property="subnet" data-sortable></oui-column>
         <oui-column data-title="'iplb_vrack_list_col_nat_ip' | translate" data-property="natIp" data-sortable></oui-column>
         <oui-column data-title="'iplb_vrack_list_col_farms' | translate">
-            <cui-labels data-items="$row.farmId">{{ $item.displayName || $item.farmId }}</cui-labels>
+            <span class="oui-chip" data-ng-repeat="farm in $row.farmId">{{ farm.displayName || farm.farmId }}</span>
         </oui-column>
         <oui-action-menu align="end" compact>
```

Comment:
Could you please add missing `data-` prefix?

Reply:
data- prefix is already there for the angular attributes... class is a native attribute and hence does not have the prefix. Am I missing something else?

Reply:
@ganeshkumar1989 you're right.

Here I'm targeting the `align` and `compact` attributes which are attributes related to the `<oui-action-menu>` component (according to the API table from the documentation http://master.ui-kit.ovh/#!/oui-angular/action-menu).